zip-0244.rst

```diff
@@ -600,6 +600,10 @@ The overall structure of the hash is as follows::
 Each node written as ``snake_case`` in this tree is a BLAKE2b-256 hash of authorizing
 data of the transaction.
 
+For transaction versions before v5, a placeholder value consisting of 32 bytes of
+``0xFF`` is used in place of the authorizing data commitment. This is only used in
+the tree committed to by ``hashAuthDataRoot``, as defined in `Block Header Changes`_.
```

Comment:
I don't personally think this separation is necessary, as it only helps the `hashAuthDataRoot` to commit to the exact number of transactions in the block, but that commitment has no value on its own, and `hashMerkleRoot` already commits to the number of transactions in the block. However, I don't see any harm in doing this, and once v4 transactions are removed from consensus in a future NU, it won't matter anymore.

I checked that v4 transactions are prohibited from the wider ZIP 239 network messages, so the "only used" wording here is correct.